### ISSUE TITLE
Flaky contract kit tests

### ISSUE THREAD

**Issue by asaj** [celo-org/celo-monorepo#1773]
### Expected Behavior

ContractKit tests pass always.

### Current Behavior

Occasionally, all tests fail. Likely something to do with the setup
```
    console.error ../../node_modules/jest-jasmine2/build/jasmine/Env.js:289
      Unhandled error
    console.error ../../node_modules/jest-jasmine2/build/jasmine/Env.js:290
      TypeError: Cannot read property 'pop' of undefined
          at c.x._updateNode (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:145275)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:143284
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:146795
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:392
          at c (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:5407)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:317
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:392
          at c (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:5407)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:317
          at e (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:146267)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:146726
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:143836
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:2:94440
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:392
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:5626
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:317
    console.error ../../node_modules/jest-jasmine2/build/jasmine/Env.js:289
      Unhandled error
    console.error ../../node_modules/jest-jasmine2/build/jasmine/Env.js:290
      TypeError: Cannot read property 'pop' of undefined
          at c.x._updateNode (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:145275)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:143284
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:146795
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:392
          at c (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:5407)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:317
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:392
          at c (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:5407)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:317
          at e (/home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:146267)
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:146726
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:143836
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:2:94440
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:392
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:5626
          at /home/circleci/app/node_modules/@celo/ganache-cli/build/ganache-core.node.cli.js:25:317

  ● SortedOracles Wrapper › #report › when reporting from a whitelisted Oracle › should be able to report a rate

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

      at mapper (../../node_modules/jest-jasmine2/build/queueRunner.js:25:45)

  ● SortedOracles Wrapper › #report › when reporting from a whitelisted Oracle › should be able to report a rate

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.
```

**Comment by H34D** [comment]
Even if they not fail completely they still fail partially:

```
Test Suites: 7 failed, 9 passed, 16 total
Tests:       51 failed, 49 passed, 100 total
Snapshots:   0 total
Time:        156.168s
```

on current master

**Comment by H34D** [comment]
closed with #2150
